src/app/components/Timestamp/index.jsx

```diff
@@ -26,12 +33,12 @@ const StyledTimestamp = styled.span`
 `;
 
 const Timestamp = ({ timestamp }) => {
-  if (!timestamp) {
+  const formattedTimestamp = formatTimestamp(timestamp);
+
+  if (formattedTimestamp === null) {
```

Comment:
I'd suggest still using `if (!formattedTimestamp)` since that will capture the cases for it being null or undefined.